.golangci.yml

```diff
@@ -10,10 +10,14 @@ linters:
     - nestif #https://github.com/nakabonne/nestif
     - exportloopref #https://github.com/kyoh86/exportloopref
     - bodyclose #https://github.com/timakin/bodyclose
+    - goconst #https://github.com/jgautheron/goconst
     - errcheck #https://github.com/kisielk/errcheck
     - stylecheck #https://github.com/dominikh/go-tools/tree/master/stylecheck
     - revive #golint is deprecated and golangci-lint recommends to use revive instead https://github.com/mgechev/revive
     #other deprecated lint libraries: maligned, scopelint, interfacer
+    - gocritic
```

Comment:
Looking at the issues that gocritic is listing bellow `hugeParam` is nice! It agrees with the type of refactoring we did previously around ListOptions and pointers